src/ur5e_arm.cpp

```diff
@@ -128,6 +128,8 @@ void reportRobotProgramState(bool program_running) {
     VIAM_SDK_LOG(info) << "\033[1;32mUR program running: " << std::boolalpha << program_running << "\033[0m";
 }
 
+enum class TrajectoryStatus : int8_t { RUNNING = 1, CANCELLED = 2, STOPPED = 3 };
```

Comment:
- Since this is used only internally to this file, you can and should put the definition for this in the unnamed namespace above.
- You don't actually need to pick an integral type here, the compiler will pick a small one for you.
- Let's switch over to new style constants here. I'd call these `k_running`, `k_cancelled`, and `k_stopped`.